exovetter/utils.py

```diff
@@ -0,0 +1,423 @@
+"""Utility functions."""
+
+import sys
+import warnings
+
+import numpy as np
+
+__all__ = ['sine', 'mark_transit_cadences', 'WqedLSF']
+
+
+def sine(x, order, period=1):
+    """Sine function for SWEET vetter."""
+    w = 2 * np.pi / period
+    if order == 0:
+        return np.sin(w * x)
+    elif order == 1:
+        return np.cos(w * x)
+    else:
+        raise ValueError("Order should be zero or one")
+
+def estimate_scatter(flux):
+    """Estimate the typical scatter in a lightcurve.
+
+    Uses the same method as Marshall (Mullally et al 2017)
+
+    Inputs:
+    ----------
+    flux
+        (np 1d array). Flux to measure scatter of. Need not have
+        zero mean.
+
+    Returns:
+    ------------
+    (float) scatter of data in the same units as in the input ``flux``
+
+
+    Notes:
+    ----------
+    Algorithm is reasonably insensitive to outliers. For best results
+    uses outlier rejection on your lightcurve before computing scatter.
+    Nan's and infs in lightcurve will propegate to the return value.
```

Comment:
Are you sure it will actually propagate? The first thing you did is only select the finite values in flux. Do you actually mean the opposite?

Also, some grammatical corrections:

```suggestion
    Algorithm is reasonably insensitive to outliers. For best results,
    use outlier rejection on your lightcurve before computing the scatter.
    NaN's and inf's in lightcurve will propagate to the return value.
```